src/includes/set-release/apple.mdx

```diff
@@ -2,15 +2,15 @@
 import Sentry
 
 SentrySDK.start { options in
-    options.releaseName = "1.0.0"
+    options.releaseName = "io.example.MyApp@1.0.0"
```

Comment:
it's also done automatically for Apple.

Reply:
The point of this page is to show how this can be done manually, the text above states that.

Reply:
true, I just looked at the diff, so no text was shown, sry about that.